.github/workflows

```diff
@@ -0,0 +1,70 @@
+name: Linters
+
+on: pull_request
```

Comment:
- [ ] Your linters are not working. Kindly fix your linters because linters prevent your from making those errors in your code which can break your application 🤔